src/cmd.js

```diff
@@ -258,6 +291,71 @@ function action(actionToDispatch) {
   });
 }
 
+function clearTimeoutCmd(timerId) {
+  return run(clearTimeout, { args: [timerId] });
+}
+
+function clearIntervalCmd(timerId) {
+  return run(clearInterval, { args: [timerId] });
+}
+
+function setTimeoutCmd(nestedCmd, delayMs, options = {}) {
+  return delay(nestedCmd, delayMs, options, false);
+}
+
+function setIntervalCmd(nestedCmd, delayMs, options = {}) {
+  return delay(nestedCmd, delayMs, options, true);
+}
+
+function delay(nestedCmd, delayMs, options, isRepeating) {
+  if (process.env.NODE_ENV !== 'production') {
+    const name = isRepeating ? 'Cmd.setInterval' : 'Cmd.setTimeout';
```

Comment:
i feel like if we need the name here, we should pass it in. If we rename the cmd, we shouldn't have to look here to change that.

Reply:
The name is passed in now.